features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/multi-dc/OGG/oracle/apimgt/tables.sql

```diff
@@ -1380,13 +1402,13 @@ CREATE TABLE AM_SUBSCRIPTION (
 
 --TODO: Have to add ON UPDATE CASCADE for the FOREIGN KEY(APPLICATION_ID) relation
 CREATE TABLE AM_APPLICATION_KEY_MAPPING (
-    UUID VARCHAR2(512),
+    UUID VARCHAR2(100),
     APPLICATION_ID INTEGER,
     CONSUMER_KEY VARCHAR2(512),
     KEY_TYPE VARCHAR2(512) NOT NULL,
     CREATE_MODE VARCHAR2(30) DEFAULT 'CREATED',
     STATE VARCHAR2(30) NOT NULL,
-    KEY_MANAGER VARCHAR2(512),
+    KEY_MANAGER VARCHAR2(100),
```

Comment:
Isn't this length reduction affecting migrations? 
Or was this handled via some other approach?

Reply:
This was added to sync the script with changes in https://github.com/wso2/carbon-apimgt/pull/8831. This change was already there in the default sql scripts but had not been synced to multi-dc scripts.